packages/puppeteer-core/src/bidi/core/BrowsingContext.ts

```diff
@@ -601,6 +601,22 @@ export class BrowsingContext extends EventEmitter<{
     );
   }
 
+  @throwIfDisposed<BrowsingContext>(context => {
+    // SAFETY: Disposal implies this exists.
+    return context.#reason!;
+  })
+  async setScreenOrientationOverride(
+    screenOrientation: Bidi.Emulation.ScreenOrientation | null,
+  ): Promise<void> {
+    await this.userContext.browser.session.send(
```

Comment:
Nit: lets use the helper getter.
```suggestion
    await this.#session.send(
```